web/frontend/scripts/copy-prime-assets.js

```diff
@@ -1,6 +1,5 @@
 /* eslint-disable @typescript-eslint/no-var-requires */
-/* eslint-disable unicorn/prefer-module */
-const fs = require('node:fs');
+import fs from 'node:fs';
```

Comment:
I've aligned the package.json for RDK to use `type: module` like our other repos.